assets/js/lesson_3/math_operations.js

```diff
@@ -0,0 +1,14 @@
+const firstInt = +prompt('Enter first number');
+const secondInt = +prompt('Enter second number');
+
+if (!isNaN(firstInt) && !isNaN(secondInt)) {
+    const add = firstInt + secondInt;
+    const sub = firstInt - secondInt;
+    const mult = firstInt * secondInt;
+    const div = firstInt / secondInt; 
+    
+    const result = `Result \n${firstInt}+${secondInt}=${add} \n${firstInt}-${secondInt}=${sub} \n${firstInt}*${secondInt}=${mult} \n${firstInt}/${secondInt}=${div}`
```

Comment:
You can use separate lines with template strings. It will be easier to read
```suggestion
    const result = `Result \n
    ${firstInt}+${secondInt}=${add} \n
    ${firstInt}-${secondInt}=${sub} \n
    ${firstInt}*${secondInt}=${mult} \n
    ${firstInt}/${secondInt}=${div}`
```